README.md

```diff
@@ -65,82 +64,21 @@ image_with_preds = overlay_predictions(predictions, image)
 image_with_preds.save("image.jpg")
 ```
 
-### Store API Credentials
-
-Here are the three ways to configure your API Key and API Secret, ordered by the priority in which they are loaded:
-
-1. Pass them as function parameters.
-
-2. Set them as environment variables. For example: `export LANDINGAI_API_KEY=...`, `export LANDINGAI_API_SECRET=...`.
-
-3. Store them in an `.env` file under your project root directory. For example, here is a set of credentials in an `.env` file:
-
-```
-   LANDINGAI_API_KEY=v7b0hdyfj6271xy2o9lmiwkkcb12345
-   LANDINGAI_API_SECRET=ao6yjcju7q1e6u0udgwrgknhrx6m4n1o48z81jy6huc059gne047l4fq312345
-```
-
 ## Run Examples Locally
 
 All the examples in this repo can be run locally.
 
 To give you some guidance, here's how you can run the `rtsp-capture` example locally in a shell environment:
 
 1. Clone the repo to local: `git clone https://github.com/landing-ai/landingai-python.git`
-2. Install the library: `poetry install --with examples` (Note: See below for how to install `poetry`)
+2. Install the library: `poetry install --with examples` (Note: See [Developer Guide](https://landing-ai.github.io/landingai-python/landingai.html#developer-guide) for how to install `poetry`)
 3. Activate the virtual environment: `poetry shell`
 4. Run: `python landingai-python/examples/capture-service/run.py`
 
-## Build the `landingai` Library Locally (for Contributors)
-
-Most of the time you won't need to build the library because it is included in this repository and also published to PyPI. But if you'd like to contribute to this repository, follow the instructions below.
-
-### Prerequisite - Install Poetry
-
-> `landingai` uses `Poetry` for packaging and dependency management. If you want to build it from source, you have to install Poetry first. To see all possible options, go to [the official guide](https://python-poetry.org/docs/#installation).
-
-For Linux, macOS, and Windows (WSL):
-
-```
-curl -sSL https://install.python-poetry.org | python3 -
-```
-
-Note: You can switch to use a different Python version by specifying the Python version:
-
-```
-curl -sSL https://install.python-poetry.org | python3.10 -
-```
-
-Or run the following command after you install Poetry:
-
-```
-poetry env use 3.10
-```
-
-### Install All Dependencies
-
-```bash
-poetry install --all-extras
-```
-
-### Run Tests
-
-```bash
-poetry run pytest tests/
-```
-
-### Activate the virtualenv
-
-```bash
-poetry shell
-```
-
 ## Documentation
 
 -  [Landing AI Python Library API Reference](https://landing-ai.github.io/landingai-python/landingai.html)
-
+-  [Landing AI Python Library User Guide](https://landing-ai.github.io/landingai-python/landingai.html#user-guide)
+-  [Landing AI Python Library Developer Guide](https://landing-ai.github.io/landingai-python/landingai.html#developer-guide)
 -  [Landing AI Suport Center](https://support.landing.ai/)
```

Comment:
Line 83: Change "Suport" to "Support" (that was my typo, sorry!)

Reply:
good catch, done: https://github.com/landing-ai/landingai-python/commit/ca9866fdeb2c9d603c1994813977a45f4cfe12c5